### PR TITLE
chore: add hardforks and enable_vrf_adjustion_block option to it. some  small fixes

### PR DESCRIPTION
**Pull request by kstdl**
## Purpose

Add ability to disable vrf params adjusting on start and enable it after with hardfork functionality

### CODE REVIEW

File: libraries/core_libs/consensus/src/dag/sortition_params_manager.cpp [hunk 1]
```diff
@@ -173,13 +187,14 @@ std::optional<SortitionParamsChange> SortitionParamsManager::calculateChange(uin
   }
 
   const int32_t change = getChange(period, average_dag_efficiency);
-  config_.vrf += change;
+  VrfParams vrf = config_.vrf;
+  vrf += change;
```
Comment (JakubFornadel) [inline libraries/core_libs/consensus/src/dag/sortition_params_manager.cpp:191]: just making sure you wanted to change this from saving the change is class member into not saving it ?

**Comment by MatusKysel** [comment]
maybe you should also increase version or genesis timestamp so when we merge it, it will be ready to deploy

**Review by MatusKysel — APPROVED** [review]

File: libraries/core_libs/consensus/src/dag/sortition_params_manager.cpp (hunk 1 above)
Comment (kstdl) [reply libraries/core_libs/consensus/src/dag/sortition_params_manager.cpp:191]: It moved here https://github.com/Taraxa-project/taraxa-node/pull/1306/files#diff-740ffa012af5a6c39db38b0cba5ee86455365dd3397a774838fd38b043e5a8c6R156. I noticed and thought that method with a name "calculateChange" shouldn't change anything. So I moved it and made this method const

Comment (JakubFornadel) [reply libraries/core_libs/consensus/src/dag/sortition_params_manager.cpp:191]: ah I see, ok

**Pull request closed without merging**